approval_polls/templates/approval_polls/my_polls.html

```diff
@@ -0,0 +1,52 @@
+{% extends 'base.html' %}
+
+{% block content %}
+  <div  class='row-fluid top-buffer'>
+      <div class='col-md-12'>
+          <div class='well well-sm'>
+            Polls created by {{ user.username }}:
+        </div>
```

Comment:
Instead of lines 6-8, use `<h3> My Polls </h3>`. Also, could you indent the html properly?